VideoLocker/res/layout/fragment_my_friends_course_list_tab.xml

```diff
@@ -39,12 +39,15 @@
             android:textAllCaps="true"
             android:textStyle="bold"
             android:drawableLeft="@drawable/ic_facebookwhite"
+            android:drawableStart="@drawable/ic_facebookwhite"
             android:drawablePadding="12dp"
             android:paddingLeft="12dp"
+            android:paddingStart="12dp"
             android:layout_marginRight="8dp"
             android:layout_marginLeft="8dp"
             android:layout_marginTop="12dp"
```

Comment:
I know.  but it is a good idea to add that.  
because we should not have unnecessary assumption, to depends on that additional / hidden logic.